ui/src/warp_runner/commands.rs

```diff
@@ -238,3 +246,42 @@ async fn multipass_initialize_friends(
     };
     Ok(ret)
 }
+
+async fn raygun_initialize_conversations(
+    convs: &[raygun::Conversation],
+    account: &Account,
+    messaging: &mut Messaging,
+) -> Result<(state::Identity, HashMap<Uuid, chats::Chat>), Error> {
+    let own_identity = account.get_own_identity().await?;
+    let mut all_chats = HashMap::new();
+    for conv in convs {
+        match conversation_to_chat(conv, account, messaging).await {
+            Ok(chat) => {
+                let _ = all_chats.insert(chat.id, chat);
+            }
+            Err(e) => {
+                logger::error(&format!("failed to convert conversation to chat: {}", e));
+            }
+        };
+    }
+    Ok((state::Identity::from(own_identity), all_chats))
+}
+
+async fn raygun_remove_2person_conv(
+    recipient: DID,
+    messaging: &mut Messaging,
+) -> Result<(), Error> {
+    match messaging.list_conversations().await {
+        Ok(convs) => {
+            for conv in convs {
+                // check if conversation should be deleted
+                // only consider conversations with 2 participants
+                if conv.recipients().len() == 2 && conv.recipients().contains(&recipient) {
```

Comment:
Not sure if this matters but you could check the conversation type and if its `ConversationType::Direct` (which would only contains 2 recipients) instead. If its `ConversationType::Group` it can be expected to have more than 2 recipients.